contentgrid-appserver-query-engine-impl-jooq/build.gradle

```diff
@@ -12,7 +13,18 @@ dependencies {
     api 'org.springframework.boot:spring-boot-starter-jooq'
     api 'com.contentgrid.thunx:thunx-model'
     implementation 'com.fasterxml.uuid:java-uuid-generator'
+    compileOnly 'org.projectlombok:lombok:1.18.30'
+    testCompileOnly 'org.projectlombok:lombok:1.18.30'
+    annotationProcessor 'org.projectlombok:lombok:1.18.30'
+    testAnnotationProcessor 'org.projectlombok:lombok:1.18.30'
```

Comment:
Any platform-based way to have this match the lombok version we use in the rest of the project?